### DISCUSSION TITLE
Do emissions calculations account for exports?

### DISCUSSION THREAD

**Discussion by DuncanCampbell** [NREL/REopt-API-Analysis#106] [6 - Emissions Questions] [answered]
Do the emissions calculations performed by Re-Opt account for the emissions value of exports?

**Comment by adfarth** [comment, marked as answer]
By default, exported electricity is assumed to offset emissions from the grid. There is a section under the "Renewable Energy & Emissions" accordion (Advanced inputs), called "Treatment of Exported Electricity" where you can change that default. The same applies for whether exported electricity counts towards renewable energy accounting (default is yes, and users can change this).
